apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html

```diff
@@ -1129,3 +1129,12 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
     </div>
 </ng-template>
 <!-- Tourist scheme template block end-->
+<!--Advance receipt adjusted remove confirmation modal -->
+<div bsModal #advanceReceiptRemoveConfirmationModal="bs-modal" class="modal fade" role="dialog" [config]="{keyboard:true}" tabindex="-1">
```

Comment:
`delete-template-confirmation-modal` should not be in the DOM if the modal is not shown consider adding `*ngIf` to the component with condition `advanceReceiptRemoveConfirmationModal.isShown`